fossology_doxygen.conf

```diff
@@ -55,7 +55,7 @@ PROJECT_NAME           = "FOSSology"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         =
+PROJECT_NUMBER         = 3.4.0-rc2
```

Comment:
3.4.0? or maybe 3.5.0-rc1?

Reply:
I think it will be better not to hard code the version as it has to be changed on every release and travis anyways [gets it from git](https://github.com/fossology/fossology/blob/master/utils/deploy-pages.sh#L36)